src/training/training.h

```diff
@@ -16,6 +16,7 @@ template <class ModelWrapper>
 class Train : public ModelTask {
 private:
   Ptr<Options> options_;
+  void installCustomSignalHandlers();
```

Comment:
Is it installing multiple signal handlers?

Reply:
Potentially in the  future. I prefer to keep API changes rare.

Reply:
Please rename to `installEarlyExitSignalHandler()`

Reply:
I strongly disagree on this one. installCustomSignalHandlers() is like parseOptions() a generic concept: install whatever custom signal handlers may be specified in the options. Either my pull request goes in as-is now, or I'll withdraw it. @emjotde